src/styles/components/statusbar.less

```diff
@@ -0,0 +1,49 @@
+.status-bar {
+  display: inline-flex;
+  background-color: @gray;
+  margin-right: @base-spacing-unit * 2/3;
```

Comment:
When you use `@base-spacing-unit`, make sure to create styles for all screens, since we have different variables for different screens: `@base-spacing-unit`, `@base-spacing-unit-screen-mini`, `@base-spacing-unit-screen-small`, etc.

See here for an example: https://github.com/dcos/dcos-ui/blob/master/src/styles/components/filter-bar.less